rskj-core/src/main/java/org/ethereum/core/BlockFactory.java

```diff
@@ -55,6 +55,10 @@ public synchronized static BlockFactory getInstance() {
         return INSTANCE;
     }
 
+    public Block cloneBlockForModification(Block block) {
```

Comment:
I chose that higher level name because I think that sealing/unsealing a block is a terrible mix of concerns. In the future we should return a `ModifiableBlock` instance or similar...